include/net/xia_ether.h

```diff
@@ -0,0 +1,36 @@
+#ifndef _NET_XIA_ETHER_H
+#define _NET_XIA_ETHER_H
+
+#include <linux/netdevice.h>
+#include <linux/etherdevice.h>
+#include <linux/err.h>
+#include <linux/netlink.h>
+#include <linux/rwlock.h>
+#include <linux/spinlock.h>
+
+/* Ethernet Principal. */
+#define XIDTYPE_ETHER (__cpu_to_be32(0x22))
+
+#ifdef __KERNEL__
+
+/* ETHER's virtal XID type. */
+int ether_vxt __read_mostly = -1;
+
+/* ETHER context. */
+struct xip_ether_ctx
+{
+	struct net          *net;
+	struct xip_ppal_ctx ctx;
+};
+
+static inline struct xip_ether_ctx *ctx_ether(struct xip_ppal_ctx *ctx)
+{
+	return likely(ctx)
+		? container_of(ctx, struct xip_ether_ctx, ctx)
+		: NULL;
+}
+
+extern int ether_vxt;
+
+#endif /* __KERNEL__ */
+#endif /* _NET_XIA_ETHER_H */
```

Comment:
Hi @sauravkumar2014,

Isn't your desktop/laptop running Linux? What're these characters at the very end of the files? You may want to take a look at the command dos2unix:

https://linux.die.net/man/1/dos2unix